source/tests/system/nidcpower_driver_api_tests.cpp

```diff
@@ -140,6 +187,48 @@ TEST_F(NiDCPowerDriverApiTest, NiDCPowerReset_SendRequest_ResetCompletesSuccessf
   expect_api_success(response.status());
 }
 
+TEST_F(NiDCPowerDriverApiTest, ConfigureOutputFunctionAndVoltageLevel_ConfiguresSuccessfuly)
+{
+  const char* channel_name = "0";
+  ViReal64 expected_voltage_level = 3.0;
+  configure_output_function(channel_name, dcpower::OutputFunction::OUTPUT_FUNCTION_NIDCPOWER_VAL_DC_VOLTAGE);
```

Comment:
The dcpower::OutputFunction::OUTPUT_FUNCTION_NIDCPOWER_VAL_DC_VOLTAGE can be extracted to a local variable something like expected_output_function and used in the assert statements below also

Reply:
Done.